Yoast/wordpress-seo
# Refrain from using the Options framework in loading removed option

As the option does not exist anymore, the values cannot be requested via the Options framework.

## Summary

This PR can be summarized in the following changelog entry:

* Fixes a bug where the breadcrumb-settings are not migrated correctly.

## Test instructions

This PR can be tested by following these steps:

* Checkout 6.3.1 (or lower)
* Adjust breadcrumb settings
* Checkout master (or 7.0 tag)
* See the settings are migrated as expected

## Quality assurance


Fixes #9121
